services/core-api/app/api/compliance/models/compliance_article.py

```diff
@@ -27,4 +26,8 @@ def __repr__(self):
 
     @classmethod
     def find_by_compliance_article_id(cls, id):
-        return cls.query.filter_by(compliance_article_id=id).first()
\ No newline at end of file
+        return cls.query.filter_by(compliance_article_id=id).first()
+
+    @classmethod
+    def get_active(cls):
+        return cls.query.all()
```

Comment:
Is there a reason this get active is different than all the others?

Reply:
i need all classes to have a get_active member to work with my global static content fetcher (i do now realize how similar this is to your static_data loading for marshmallow/swagger, but not sure they can be combined easier). This model didn't have a fetch all helper at all was being queried directly. since there isn't one all of them are active, we can add an active indicator and filter and this still works as intended.